From 16eb8ea0a705c3ed1135e9a82edf951a7fa38093 Mon Sep 17 00:00:00 2001
From: remittor <remittor@gmail.com>
Date: Fri, 25 Mar 2016 12:43:42 +0300
Subject: [PATCH] bionic: Get rid of "has text relocations" warn messages

---
 linker/linker.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/linker/linker.cpp b/linker/linker.cpp
index cd10131..2cff618 100644
--- a/linker/linker.cpp
+++ b/linker/linker.cpp
@@ -3016,9 +3016,7 @@ bool soinfo::link_image(const soinfo_list_t& global_group, const soinfo_list_t&
 #if !defined(__i386__) // ffmpeg says that they require text relocations on x86
     if (get_application_target_sdk_version() != __ANDROID_API__
         && get_application_target_sdk_version() > 22) {
-      PRINT("%s: has text relocations", get_realpath());
       DL_ERR("%s: has text relocations", get_realpath());
-      return false;
     }
 #endif
     // Make segments writable to allow text relocations to work properly. We will later call
-- 
1.9.5.msysgit.0

