From 4495f9a3a6b3f76b4987779898b40766d4909176 Mon Sep 17 00:00:00 2001
From: remittor <remittor@gmail.com>
Date: Wed, 16 Dec 2015 13:10:51 +0300
Subject: [PATCH] Add hook functions to post_process_props.py

---
 tools/post_process_props.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/post_process_props.py b/tools/post_process_props.py
index 5193d4b..e5d8966 100755
--- a/tools/post_process_props.py
+++ b/tools/post_process_props.py
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 import os, sys
+from post_process_props_hook import *
 
 # Usage: post_process_props.py file.prop [blacklist_key, ...]
 # Blacklisted keys are removed from the property file, if present
@@ -129,8 +130,10 @@ def main(argv):
 
   if filename.endswith("/build.prop"):
     mangle_build_prop(properties, extraargs)
+    mangle_build_prop_hook(properties, extraargs)
   elif filename.endswith("/default.prop"):
     mangle_default_prop(properties)
+    mangle_default_prop_hook(properties)
   else:
     sys.stderr.write("bad command line: " + str(argv) + "\n")
     sys.exit(1)
-- 
1.9.5.msysgit.0

